DataCollector/ProviderDataCollector.php

```diff
@@ -84,6 +84,11 @@ public function unserialize($data)
         $this->providers = $unserialized['providers'];
     }
 
+    public function reset()
```

Comment:
Can you explain please?

Reply:
this is required for Symfony 4, actually , I don't why it need this method.